release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -338,6 +338,17 @@ For more information about adding devices to a volume group, see xref:../storage
 [id="ocp-4-16-olm_{context}"]
 === Operator lifecycle
 
+[id="ocp-4-16-olm-improved-status-conditions_{context}"]
+==== Improved status condition messages and deprecation notices for cluster extensions in {olmv1-first} (Technology Preview)
+
+With this release, {olmv1} displays following status condition messages for installed cluster extensions:
+
+* Specific bundle name
+* Installed version
+* Channel information
```

Comment:
Hi, I didn't find the `channel` info in the `status.conditions` fields, as follows, or am I missing something? Thanks!
```console
jiazha-mac:~ jiazha$ oc get clusterextension pipelines-operator -o yaml
apiVersion: olm.operatorframework.io/v1alpha1
kind: ClusterExtension
metadata:
  creationTimestamp: "2024-06-18T02:59:21Z"
  generation: 1
  name: pipelines-operator
  resourceVersion: "128267"
  uid: 55a7d40d-e1a9-49f1-a9e1-679ffe5af116
spec:
  channel: latest
  installNamespace: test
  packageName: openshift-pipelines-operator-rh
  upgradeConstraintPolicy: Enforce
status:
  conditions:
  - lastTransitionTime: "2024-06-18T02:59:30Z"
    message: resolved to "registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:dd3d18367da2be42539e5dde8e484dac3df33ba3ce1d5bcf896838954f3864ec"
    observedGeneration: 1
    reason: Success
    status: "True"
    type: Resolved
  - lastTransitionTime: "2024-06-18T02:59:40Z"
    message: installed from "registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:dd3d18367da2be42539e5dde8e484dac3df33ba3ce1d5bcf896838954f3864ec"
    observedGeneration: 1
    reason: Success
    status: "True"
    type: Installed
  - lastTransitionTime: "2024-06-18T02:59:30Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: Deprecated
  - lastTransitionTime: "2024-06-18T02:59:30Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: PackageDeprecated
  - lastTransitionTime: "2024-06-18T02:59:30Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: ChannelDeprecated
  - lastTransitionTime: "2024-06-18T02:59:30Z"
    message: ""
    observedGeneration: 1
    reason: Deprecated
    status: "False"
    type: BundleDeprecated
  installedBundle:
    name: openshift-pipelines-operator-rh.v1.14.4
    version: 1.14.4
  resolvedBundle:
    name: openshift-pipelines-operator-rh.v1.14.4
    version: 1.14.4
```

Reply:
```
    reason: Deprecated
    status: "False"
    type: ChannelDeprecated
```

And I explain the status conditions here: https://github.com/openshift/openshift-docs/pull/77274/files#diff-57d7e8d28879735eab92932f9cc5f093124acaf00427fc24c2ce84e78e186d99R239-R247

Reply:
@jianzhangbjz: Thank you for the comment. PTAL at my answer and let me know if this addresses your concern.

Reply:
Yeah, but I guess these are Deprecated type as you described in the following(line 350): `* Deprecation notices for packages, channels, and bundles`.

Reply:
Got it. Sorry, I misunderstood the first time. I will remove the line about channel information. Thank you!

Reply:
@jianzhangbjz: Addressed. Thank you for being patient with me. :)

Reply:
Thanks!